docs/modules/ROOT/partials/recovery/recover-etcd.adoc

```diff
@@ -0,0 +1,147 @@
+[abstract]
+--
+ifeval::["{provider}" == "cloudscale"]
+Steps to recover etcd on an OpenShift 4 cluster on https://www.cloudscale.ch/[cloudscale.ch].
+endif::[]
+ifeval::["{provider}" == "exoscale"]
+Steps to recover etcd on an OpenShift 4 cluster on https://www.exoscale.com/[Exoscale].
+endif::[]
+--
+
+[WARNING]
+====
+Restoring to a previous cluster state is a destructive and destablizing action to take on a running cluster. This should only be used as a last resort.
+
+If you are able to retrieve data using the Kubernetes API server, then etcd is available and you shouldn't restore using an etcd backup.
+====
+
+== Starting situation
+
+ifeval::["{provider}" == "cloudscale"]
+* You have an OpenShift 4 cluster on cloudscale.ch
+endif::[]
+ifeval::["{provider}" == "exoscale"]
+* You have an OpenShift 4 cluster on Exoscale
+endif::[]
+* One of the following scenarios is true:
+** The cluster has lost the majority of control plane hosts (quorum loss).
```

Comment:
Maybe we should add something here that this procedure won't work if we've lost all control plane hosts. Not sure how to formulate it though.

```suggestion
** The cluster has lost the majority of its control plane hosts (quorum loss).
```